templates/registration/login.html

```diff
@@ -33,9 +32,12 @@
       <!-- <Input type="submit" value="login" />
       <Input type="hidden" name="next" value="{{ next }}" /> -->
       <button> Submit</button>
-  </center>
-  </form>
   
+  </form>
+  <Button onclick="window.location.href = 'https://oauth.oit.duke.edu/oauth/authorize.php?client_id=hyposoft-ev2&client_secret=X8ctUJYCH33Z31wAbz6*v4Ezz%MqWPrdMsSxIdB2eLA2+B@=d@&redirect_uri=http%3A%2F%2F0.0.0.0%3A5000%2F&response_type=token&state=1129&scope=basic'">
+    Click here to sign in with Duke NetID.
+  </Button>
+</center>
```

Comment:
can't have this exposed publically! can you factor the URL out into an environment variable?

Reply:
ohhh yes i can hehe oops